packages/sdk/src/solana/README.md

```diff
@@ -0,0 +1,69 @@
+<p align="center">
+<br />
+<a href="https://thirdweb.com"><img src="https://github.com/thirdweb-dev/js/blob/main/packages/sdk/logo.svg?raw=true" width="200" alt=""/></a>
+<br />
+</p>
+<h1 align="center">thirdweb Solana SDK</h1>
+<p align="center">
+<a href="https://discord.gg/thirdweb"><img alt="Join our Discord!" src="https://img.shields.io/discord/834227967404146718.svg?color=7289da&label=discord&logo=discord&style=flat"/></a>
+
+</p>
+<p align="center"><strong>thirdweb's Solana SDK for Browser, Node and React Native</strong></p>
+<br />
+
+## Installation
+
+Install the latest version of the SDK with either `npm` or `yarn`:
+
+```shell
+npm install @thirdweb-dev/solana
+```
+
+```shell
+yarn add @thirdweb-dev/solana
+```
```

Comment:
this is not correct anymore

Reply:
yea we just copy pasted from old version good catch

Reply:
do yk why build is broken

Reply:
no but looking at it